aries-site/src/components/cards/PreviewCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from 'grommet';
+
+export const PreviewImageCard = ({ children, ...rest }) => {
+  return (
+    <Box
+      height="small"
+      round="xsmall"
+      overflow="hidden"
+      style={{ position: 'relative' }}
+      {...rest}
+    >
+      {children}
+    </Box>
+  );
+};
```

Comment:
Makes the background prop a bit more explicit that it is a primary prop to be manipulated.
```suggestion
export const PreviewImageCard = ({ background, children, ...rest }) => {
  return (
    <Box
      background={background}
      height="small"
      round="xsmall"
      overflow="hidden"
      style={{ position: 'relative' }}
      {...rest}
    >
      {children}
    </Box>
  );
};
```